File: missing-number/missing-number.java
class Solution {
    public static void print(Object o) {
        System.out.println(o);
    }
    public int missingNumber(int[] nums) {
        Arrays.sort(nums);
        for(int i=0; i<nums.length; i++) {
            if(nums[i] != i)  {
                return i;
            }   
        }
        return nums.length;
    }
}

File: missing-number/missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let l = nums.length;
    const reducer = (sum, val) => sum + val;
    let sum = nums.reduce(reducer);
    return ((l * (l+1)) / 2) - sum;
};